cpskin/core/profiles/default/repositorytool.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<repositorytool>
+  <policymap purge="false">
+    <type name="directory">
+      <policy name="at_edit_autoversion" />
```

Comment:
I don't know repositorytool.xml, what's for ?

Reply:
> I don't know repositorytool.xml, what's for ?

We need this to set the versioning policy (you can set it by hand with @@types-controlpanel).
See https://github.com/plone/plone.app.versioningbehavior#usage